cpp/ql/src/experimental/Security/CWE/CWE-190/DangerousUseOfTransformationAfterOperation.qhelp

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Searching for places where data type conversion occurs late and does not affect the computational result.</p>
```

Comment:
This describes what the query is doing, but not actually what is wrong. Could you rephrase this to explain why it's bad that a type conversion occurs late and does not affect the computational result?

Reply:
how do you view this text?

```
Search for places where the result of the multiplication is subjected to explicit conversion, not the arguments. Therefore, during the multiplication period, you can lose meaningful data.
```

Reply:
That sounds more appropriate, yes. :)